login.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>로그인</title>
+  <link rel="icon" href="images/logo/favicon.ico">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css">
+  <link rel="stylesheet" href="./styles/global.css">
+  <link rel="stylesheet" href="./styles/login.css">
+</head>
+<body>
+  <div class="login_container">
+    <div class="logo_and_market_name">
+      <a href="./index.html">
+        <img src="./images/logo/panda-market-logo.png" alt="판다마켓 로고" class="logo_image">
+      </a>
+    </div>
+
+    <form class="login_form">
+      <label for="email">이메일</label>
+      <input type="email" id="email" placeholder="이메일을 입력해주세요" required>
+
+      <label for="password">비밀번호</label>
+      <div class="password_box">
+        <input type="password" id="password" placeholder="비밀번호를 입력해주세요"required>
+        <img class="hide_eye" id="first_password_eye" src="./images/eye-password-hide.svg" alt="hide_eye">
```

Comment:
alt 값은 대체 텍스트로서 사용자가 알 수 있게 작성해주시는 게 좋습니다.

**e.g.** 
```suggestion
        <img class="hide_eye" id="first_password_eye" src="./images/eye-password-hide.svg" alt="비밀번호 보기">
```